renderdoc/os/posix/apple/apple_stringio.cpp

```diff
@@ -54,9 +60,29 @@ void RemoveInputWindow(WindowingSystem windowSystem, void *wnd)
 {
 }
 
+/*
+/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/System/Library/Frameworks/Carbon.framework/Versions/Current/Frameworks/HIToolbox.framework/Versions/Current/Headers/Events.h
+
+kVK_F11                       = 0x67,
+kVK_F13                       = 0x69,
+kVK_F12                       = 0x6F,
+*/
+
 bool GetKeyState(int key)
 {
-  return false;
+  int vk = 0;
+  switch(key)
+  {
+    case eRENDERDOC_Key_F11: vk = 0x67; break;
+    case eRENDERDOC_Key_PrtScrn: vk = 0x69; break;
+    case eRENDERDOC_Key_F12: vk = 0x6F; break;
```

Comment:
Please add mappings for all the RenderDoc keys, as users can customise which keys do what using the RenderDoc API.